ios/RNSScreen.mm

```diff
@@ -980,6 +1030,81 @@ - (void)viewDidLayoutSubviews
   }
 }
 
+// Checks whether this screen has any child view controllers of type RNScreensNavigationController.
```

Comment:
Why the name here is `RNScreensNavigationController` and in code it is `RNSNavigationController`?

Reply:
Looks like the refactor tool didn't changed that when I was refactoring this class 🤔 Thanks for pointing this out!

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/1802/commits/fe776ca0bfc904f0376cc9958d7ef7661eb101cf.